Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -3284,6 +3284,7 @@ def __init__(self, howMuch, dataset):
     ('TenTau_E_15_500_pythia8_cfi', UpgradeFragment(Kby(9,100),'TenTau_15_500')),
     ('SinglePiPt25Eta1p7_2p7_cfi', UpgradeFragment(Kby(9,100),'SinglePiPt25Eta1p7_2p7')),
     ('SingleMuPt15Eta1p7_2p7_cfi', UpgradeFragment(Kby(9,100),'SingleMuPt15Eta1p7_2p7')),
+    ('SingleMuPt15Eta0_0p4_cfi', UpgradeFragment(Kby(9,100),'SingleMuPt15Eta0p_0p4')),
```

Comment:
@cms-sw/pdmv-l2 does this need to be put at the end of the list?

Reply:
Yes, if not it will break all the already existing wf numbering.

Reply:
Thanks, my bad --- @JanGerritSchulz can you please move it down? :)